mutacc_auto/files/sbatch.py

```diff
@@ -25,59 +23,81 @@ def get_timestamp():
 MAIL_FAIL = 'FAIL'
 MAIL_END = 'END'
 
-#SOME CONSTAN SBATCH
-# (OPTION, VALUE, SEPARATOR)
+#SOME DEFAULT SBATCH OPTIONS
+# (OPTION, VALUE)
 HEADER_OPTIONS = (
-    ('-A', ACCOUNT, ' '),
-    ('-n', NODES, ' '),
-    ('-t', TIME, ' '),
-    ('-J', JOBNAME, ' '),
-    ('--qos', PRIORITY, '='),
-    ('--mail-type', MAIL_FAIL, '=' ),
-    ('--mail-type', MAIL_END, '=' )
+    ('A', ACCOUNT),
+    ('n', NODES),
+    ('t', TIME),
+    ('J', JOBNAME),
+    ('qos', PRIORITY),
+    ('mail-type', MAIL_FAIL),
+    ('mail-type', MAIL_END)
 )
 
+ACTIVATE_ENVIRONMENT = "source activate"
```

Comment:
with more recent versions of conda this is now "conda activate"

Reply:
ok, I'll give some support for 'conda activate' as well!